Resources/Textures/_NF/Tips/clippy.rsi/meta.json

```diff
@@ -0,0 +1,20 @@
+{
+  "version": 1,
+  "license": "CC-BY-SA-3.0",
+  "copyright": "",
```

Comment:
not optional

Reply:
You can put me in it since I actually created it, just add my github name.

Reply:
hey I just copy paste the file

Reply:
Fix my sins, I forgot to fix it on the frontier side, just add ``"copyright": "dvir001 on github",